### PR TITLE
Raster3D-3.0-2-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**
I have added the dependencies as osdepencencies as in our cluster we prefer to do it like this. If requested I can modify it to add dependencies the "easybuild way"

this are RHEL/CentOS package names
osdependencies = [('libjpeg-turbo-devel', 'libpng-devel', 'libtiff-devel')]

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 1 out of 2
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/b28291e0614940f51ab7 for a full test report.

File: easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name = 'libjpeg-turbo'
+version = '1.3.0'
+
+homepage = 'http://sourceforge.net/libjpeg-turbo/'
+description = """libjpeg-turbo is a fork of the original IJG libjpeg which uses SIMD to accelerate baseline JPEG 
+compression and decompression. libjpeg is a library that implements JPEG image encoding, decoding and transcoding.
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb]: 1-space indentation, otherwise this looks good; @boegel : test/merge?

**Comment by fgeorgatos** [comment]
minor changes needed here, put this in REVIEW plz!

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name = 'libjpeg-turbo'
```
Comment (boegel) [inline easybuild/easyconfigs/l/libjpeg-turbo/libjpeg-turbo-1.3.0-goolf-1.4.10.eb]: missing `easyblock = 'ConfigureMake'`

**Comment by boegel** [comment]
@pescobar ping on updating this to get it merged?

**Comment by pescobar** [comment]
closing this PR to create a new one using `--new-pr`

**Comment by pescobar** [comment]
new PR here: https://github.com/hpcugent/easybuild-easyconfigs/pull/3137

**Pull request closed without merging**